Pipfile.lock

```diff
@@ -1972,11 +1975,11 @@
         },
         "pylint": {
             "hashes": [
-                "sha256:9d945a73640e1fec07ee34b42f5669b770c759acd536ec7b16d7e4b87a9c9ff9",
-                "sha256:daabda3f7ed9d1c60f52d563b1b854632fd90035bcf01443e234d3dc794e3b74"
+                "sha256:c149694cfdeaee1aa2465e6eaab84c87a881a7d55e6e93e09466be7164764d1e",
+                "sha256:dab221658368c7a05242e673c275c488670144123f4bd262b2777249c1c0de9b"
             ],
             "index": "pypi",
-            "version": "==2.12.2"
+            "version": "==2.13.5"
```

Comment:
```suggestion
            "version": "==2.12.2"
```
The problem with updating pylint is that it currently has a bug in one of its dependencies (see https://github.com/PyCQA/pylint/issues/6012 and https://github.com/tartley/colorama/issues/86) which is not yet released (see https://github.com/tartley/colorama/issues/341), so we'll have to wait until they publish a new release before we can update pylint to the next version.